.github/workflows/image-cleanup.yml

```diff
@@ -0,0 +1,28 @@
+name: Clean up PR-scoped test images
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  delete-images:
+    name: Delete PR-scoped test images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Delete Docker images
+        uses: r26d/ghcr-delete-image-action@v1.2.2
+        with:
+          owner: ${{ github.repository_owner }}
+          name: cryostat-operator
```

Comment:
You need to delete a set of images, specifically:

- `cryostat-operator`
- `cryostat-operator-bundle`
- `cryostat-operator-scorecard`

You can try matrix:

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs